src/HttpClient/HttpClientImportPogramme.php

```diff
@@ -4,26 +4,39 @@
 
 namespace App\HttpClient;
 
+use Symfony\Contracts\HttpClient\Exception\ClientExceptionInterface;
+use Symfony\Contracts\HttpClient\Exception\DecodingExceptionInterface;
+use Symfony\Contracts\HttpClient\Exception\RedirectionExceptionInterface;
+use Symfony\Contracts\HttpClient\Exception\ServerExceptionInterface;
+use Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface;
 use Symfony\Contracts\HttpClient\HttpClientInterface;
 
 class HttpClientImportPogramme
 {
-    public string $http_address = 'http://evozon-internship-data-wh.herokuapp.com/api/sport-programs';
-
-    public string $http_protocol = 'GET';
+    public const HTTP_PROTOCOL = 'GET';
```

Comment:
you don't need a constant for this

Symfony\Component\HttpFoundation\Request::METHOD_GET

and by the way, it is a method, not a protocol